packages/manager/apps/telecom/src/app/telecom/telephony/service/contact/contact.html

```diff
@@ -614,6 +614,7 @@
                             <hr />
                             <tuc-telephony-bulk-action
                                 data-service-type="all"
+                                data-filter-services="$ctrl.filterServices()"
```

Comment:
probably better like this:
```suggestion
                                data-contact-services="$ctrl.getContactServices()"
```

?

Reply:
I can't change the name because in the tuc-telephony-bulk-action module it is filterServices that is expected.